packages/template-mrt-reference-app/app/static/example.txt

```diff
@@ -0,0 +1,7 @@
+This file is used in the E2E tests to verify that Cloudfront is compressing content properly.
```

Comment:
For CloudFront to auto compress content to brotli there are a list of conditions that need to be met including:
* Viewer request must include header: Accept-Encoding: br
* content must be 1.000 bytes - 10,000,000 bytes
* content must be one of [these file types](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/ServingCompressedFiles.html#compressed-content-cloudfront-file-types)

Today, no file in the bundle path meets these criteria so we need to add this file in order to test that the bundle path supports serving data compressed with brotli!